analysis/src/npm.js

```diff
@@ -63,7 +65,7 @@ class NPM {
     return new Promise((resolve, reject) => {
       Ana.log("npm/prune");
       fs.writeFileSync('installed/package.json', '{}');
-      childProcess.exec("rm -rf installed/node_modules", function(err) {
+      childProcess.exec("rm -rf installed/node_modules installed/modules_copy", function(err) {
```

Comment:
why not use a node library to delete?

I like fs-extra, probably can replace ncp too.

Reply:
Not sure, I didn't write the original. Changed to use fs-extra.